src/validations/rules/sap.sch

```diff
@@ -230,6 +230,23 @@
                 test="oscal:prop[@ns eq 'https://fedramp.gov/ns/oscal' and @name eq 'sampling' and @value = ('yes', 'no')]">The SAP declares whether a
                 sampling method is used.</sch:assert>
 
+            <sch:assert
```

Comment:
Please add a `@see` attribute with a value of `https://github.com/GSA/fedramp-automation-guides/issues/30` to this assertion.

Reply:
Done.